examples/user_guide/Reactive_Expressions.ipynb

```diff
@@ -0,0 +1,258 @@
+    "\n",
+    "### Using Parameters\n",
+    "\n",
+    "As an example we can create reactive proxies for the `a` and `b` parameters and add them together:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "243afd62-2f10-4dd3-a264-4d9a895c3088",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "p = Parameters()\n",
+    "\n",
+    "p.param.a.reactive() + p.param.b.reactive()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f62fe083-50c9-4739-aaec-81fb31f98241",
+   "metadata": {},
+   "source": [
+    "The resulting reactive expression now reflects the result of this operation and will update automatically when one of the inputs to the operation changes, e.g. if we update parameter `a`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f71b9341-31bb-437f-90ab-3fb5da607f0a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "p.a = 5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "824dc59e-aea0-404e-b3c3-be134dffa8e0",
+   "metadata": {},
+   "source": [
+    "### Using objects"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26ad333e-0d05-4591-8492-24b641171f4d",
+   "metadata": {},
+   "source": [
+    "Note that the input to a reactive expression can be anything, e.g. we can create a `reactive` object from a static value, such as a string:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9b061295-a28c-4e13-9b8c-54f9070a0a4a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "string_template = param.reactive('Hello {name}!')\n",
+    "\n",
+    "string_template"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05af1fa3-bcea-4c55-809f-beb8e4049e90",
+   "metadata": {},
+   "source": [
+    "The `reactive` object now acts like a string so we can use the `.format` method to fill in the `string_template`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "56d8047b-b737-4b03-86a1-24b5b4d49bd3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "name = param.reactive('world')\n",
+    "\n",
+    "str_expr = string_template.format(name=name)\n",
+    "\n",
+    "str_expr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3218ab95-0261-4cd9-a4fe-6f552baafdb3",
+   "metadata": {},
+   "source": [
+    "To update the input to a `reactive` object we can use the `.set` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d4842f2f-8d17-41eb-96f0-668d0ee545d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "name.set('there')\n",
```

Comment:
And how would I `get` the underlying object?

Reply:
> Why is there not a `.value` or `object` attribute? We are used to that from Panel. Why do I need to remember a new method called `.set`?

Reactive is a proxy and as such you want its API to be as small as possible and as less likely to clash with the API of the proxied object (`set` is even questionable in this sense).

Reply:
For me it's first a question to the api. Does it have to be like that. Then it's a matter of documentation.

Reply:
I think you replied in the wrong thread?

Reply:
@maximlt's comments hit the nail on the head. The issue is that since `reactive` transparently wraps any other type of object we want to limit the API surface of `reactive` itself. I'll make a list of all the methods I added to the main description of this PR and then we can talk about whether the naming makes sense and if it's likely to clash with other objects. `value` and `object` are definitely problematic from the outset since they immediately clash, `set` was just the first thing I thought of and I'd be happy to change it if better suggestions come along.